content/docs/tasks/security/plugin-ca-cert.md

```diff
@@ -52,7 +53,7 @@ The following steps enable plugging in the certificates and key into Citadel:
 1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files:
 
     ```command
-    $ kubectl apply -f @install/kubernetes/istio-citadel-plugin-certs.yaml@
+    $ kubectl apply -f install/kubernetes/istio-citadel-plugin-certs.yaml
```

Comment:
@@

Reply:
this file is not part of the distribution.